ctru-rs/src/services/cfgu.rs

```diff
@@ -33,15 +33,15 @@ pub enum Language {
     TraditionalChinese = ctru_sys::CFG_LANGUAGE_TW,
 }
 
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[repr(u32)]
 pub enum SystemModel {
-    Model3DS = ctru_sys::CFG_MODEL_3DS,
-    Model3DSXL = ctru_sys::CFG_MODEL_3DSXL,
-    ModelNew3DS = ctru_sys::CFG_MODEL_N3DS,
-    Model2DS = ctru_sys::CFG_MODEL_2DS,
-    ModelNew3DSXL = ctru_sys::CFG_MODEL_N3DSXL,
-    ModelNew2DSXL = ctru_sys::CFG_MODEL_N2DSXL,
+    N3DS = ctru_sys::CFG_MODEL_3DS,
```

Comment:
This might be slightly confusing since sometimes I have seen `N3DS` used to mean `New 3DS`, but I suppose the fact that there is a separate `NewN3DS` should make that clear. Don't have a better suggestion really